src/controllers/due.diligence.controller.ts

```diff
@@ -5,15 +5,20 @@ import {
   FEATURE_FLAG_ENABLE_REDIS_REMOVAL,
   WHO_IS_MAKING_FILING_URL,
   ENTITY_URL,
-  ENTITY_WITH_PARAMS_URL
+  ENTITY_WITH_PARAMS_URL, PRESENTER_URL
 } from "../config";
 
 import { getDueDiligencePage, postDueDiligencePage } from "../utils/due.diligence";
 import { isActiveFeature } from "../utils/feature.flag";
 import { getUrlWithParamsToPath } from "../utils/url";
+import * as config from "../config";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
-  getDueDiligencePage(req, res, next, DUE_DILIGENCE_PAGE, WHO_IS_MAKING_FILING_URL);
+  let backLinkUrl: string = WHO_IS_MAKING_FILING_URL;
+  if (isActiveFeature(config.FEATURE_FLAG_ENABLE_REDIS_REMOVAL)) {
```

Comment:
`FEATURE_FLAG_ENABLE_REDIS_REMOVAL` already imported on line 5

Reply:
Fixed. Will be update on my next push.